CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a changelog](https://github.com/olivierlacan/keep-a-changelog).
 
 ## [Unreleased](https://github.com/idealista/superset_role/tree/develop)
+- Install new 1.0 superset version
```

Comment:
We usually use a specific format for changelogs - see https://github.com/idealista/timescaledb_role/blob/main/CHANGELOG.md

Reply:
updated changelog